### DISCUSSION TITLE
New fork for IterativeStratification

### DISCUSSION THREAD

**Discussion by crypdick** [scikit-multilearn/scikit-multilearn#261] [General]
Unfortunately, this library seems like abandon-ware. If anyone depends on `IterativeStratification`, I just created an updated version here: https://github.com/crypdick/multilabel-stratifier

Contributions welcome!

**Comment by niedakh** [comment]
Yeah, I didn't have too much time to maintain this library through the pandemic, but if you could provide a PR with the changes I'd gladly merge itd.

**Comment by ChristianSch** [comment]
There also seems to be https://github.com/trent-b/iterative-stratification
